package.json

```diff
@@ -10,17 +10,29 @@
     "preinstall": "",
     "postinstall": "gulp build",
     "start": "nf run gulp dev",
-    "test": "gulp test",
+    "test": "gulp test",    
+    "changelog": "gulp changelog",
+    "updatedocs": "gulp docs",
+    "preversion": "npm run test",
+    "version" : "npm version && npm run changelog && npm run updatedocs",
+    "postversion": "npm run commit && npm run push",
+    "commit": "git add .",
+    "push": "git push",
+    "version:major": "npm version major",
+    "version:minor": "npm version minor",
+    "version:patch": "npm version patch",
```

Comment:
@nodox What do the `version:type` commands do? Wouldn't you have to run `npm run version:patch` as opposed to `npm version patch`?